src/marshmallow_annotations/ext/namedtuple.py

```diff
@@ -0,0 +1,41 @@
+"""Specialized components for Python 3.6 NamedTuple annotations."""
+
+import marshmallow
+
+from marshmallow_annotations.scheme import AnnotationSchema, AnnotationSchemaOpts
+
+
+class NamedTupleSchemaOpts(AnnotationSchemaOpts):
+    """
+    NamedTuple specific AnnotationSchemaOpts, additionally provides:
+
+    - dump_default_fields
+    """
+    def __init__(self, meta, *args, **kwargs):
+        super().__init__(meta, *args, **kwargs)
+        self.dump_default_fields = getattr(meta, 'dump_default_fields', True)
+
+
+class NamedTupleSchema(AnnotationSchema):
+    """
+    Derived class for creating typing.NamedTuple schema with automatic
+    post-load conversion to namedtuple instances.
+    """
+    OPTIONS_CLASS_TYPE = NamedTupleSchemaOpts
+
+    @marshmallow.post_load
+    def make_namedtuple(self, data):
+        """Post load, deserialize to target namedtuple class."""
+        return self.opts.target(**data)
+
+    @marshmallow.post_dump
+    def remove_optional(self, data):
+        """Post dump, strip default fields from serialized output."""
+        if self.opts.dump_default_fields:
+            return data
+        else:
+            default_values = self.opts.converter._get_field_defaults(self.opts.target)
```

Comment:
Could we store the defaults on the opts so we don't need to look them up again? Probably not a big issue but a thought. The downside would be if a default is mutable, though you have other issues in that case as well.

Reply:
In this case, I think that's unnecessary complexity. Under the hood this call becomes a field lookup of `item._field_defaults` on the target namedtuple. This ought to have nearly equivalent performance to storing the values on opts instead.